Monika After Story/game/0utils.rpy

```diff
@@ -364,7 +371,23 @@ python early in mas_utils:
     import functools
 
     from store import mas_logging
-    mas_log = mas_logging.init_log("mas_log")
+
+    
+    def init_mas_log():
+        """
+        Initializes the MAS log, or gets it if its already init.
+
+        RETURNS: the init'd mas_log
+        """
+        if mas_logging.is_inited("mas_log"):
+            return mas_logging.get_log("mas_log")
+
+        global mas_log
+        mas_log = mas_logging.init_log("mas_log")
+        return mas_log
```

Comment:
Please, move `global` to the top of the function, that's a breaking change in py3 and we can fix it right away.

Reply:
done